compiler/rustc_passes/src/hir_id_validator.rs

```diff
@@ -1,11 +1,13 @@
-use rustc_data_structures::sync::Lock;
-use rustc_hir as hir;
-use rustc_hir::def_id::LocalDefId;
-use rustc_hir::intravisit;
-use rustc_hir::{HirId, ItemLocalId};
-use rustc_index::bit_set::GrowableBitSet;
-use rustc_middle::hir::nested_filter;
 use rustc_middle::ty::TyCtxt;
+#[cfg(debug_assertions)]
+use ::{
+    rustc_data_structures::sync::Lock,
+    rustc_hir as hir,
+    rustc_hir::def_id::LocalDefId,
+    rustc_hir::{intravisit, HirId, ItemLocalId},
+    rustc_index::bit_set::GrowableBitSet,
+    rustc_middle::hir::nested_filter,
+};
 
 pub fn check_crate(tcx: TyCtxt<'_>) {
     if tcx.sess.opts.unstable_opts.hir_stats {
```

Comment:
maybe inline this one at the one call site and then cfg out the entire module plus the `check_crate` call site?

Reply:
Sorry, I meant cfging the `mod` declaration in the parent module, not adding a nested module.